client/components/mma/switch/SwitchComplete.tsx

```diff
@@ -56,6 +86,79 @@ const extrasStyling = css`
 	}
 `;
 
+const whatHappensNextTextCss = css`
+	width: 100%;
+	margin-left: 0.5rem;
+
+	p {
+		${textSans.medium()}
+		margin-top: 0;
+		margin-bottom: ${space[2]}px;
+	}
+`;
+
+const WhatHappensNext = (props: {
+	currency: string;
+	amountPayableToday: number;
+	email: string;
+}) => {
+	return (
+		<Stack
+			cssOverrides={css`
+				svg {
+					fill: ${brand[500]};
+					flex-shrink: 0;
+				}
+			`}
+		>
+			<Heading sansSerif>What happens next?</Heading>
+			<div
+				css={css`
+					display: flex;
+					align-items: start;
+					margin-top: ${space[4]}px;
+				`}
```

Comment:
This can wait until we've got all of the components built and start doing some refactoring and tidying up, but we can probably share some of these styles with the 'what happens next' section on the previous page.